cephfs/path.go

```diff
@@ -56,3 +56,53 @@ func (mount *MountInfo) Unlink(path string) error {
 	ret := C.ceph_unlink(mount.mount, cPath)
 	return getError(ret)
 }
+
+// Link creates a new link to an existing file.
+//
+// Implements:
+//  int ceph_link (struct ceph_mount_info *cmount, const char *existing, const char *newname);
+func (mount *MountInfo) Link(oldname, newname string) error {
+	cOldname := C.CString(oldname)
+	defer C.free(unsafe.Pointer(cOldname))
+
+	cNewname := C.CString(newname)
+	defer C.free(unsafe.Pointer(cNewname))
+
+	ret := C.ceph_link(mount.mount, cOldname, cNewname)
```

Comment:
There is no validity checking of `mount.mount`, I guess if it is `nil` or otherwise invalid (unmounted?) there may be a crash.
No need to add any checking now, as none of the existing functions in this file have it either.

Reply:
That's a good point, we have this check in file.go
Should I add here too?

Reply:
You can add them in the functions you added in this PR. The other functions should get their checks with a different PR (or at least a different commit that does not add link functions).

Reply:
Agreed. This is probably good for a follow-up PR, but at the very least must be a separate patch (commit).

Reply:
Ok, will fix this in a new PR. 
Will fix the useMount() issue and update the current one before closing.